common/ECSResourcesCheck/src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import org.json.JSONObject;
+
+public class Main {
+    public static JSONObject readParam(String param){
```

Comment:
This function is only used once and basically creates a JSON object from a string. It can be deleted.

Reply:
done